### DISCUSSION TITLE
Esp8266

### DISCUSSION THREAD

**Discussion by SAQO101** [risinek/esp32-wifi-penetration-tool#95] [Ideas]
Can you make it to work in esp8266
And if possible over cli PLSSSSSSSS

**Comment by risinek** [comment]
ESP8266 has limitations in this area. Check [similar projects mentioned in README](https://github.com/risinek/esp32-wifi-penetration-tool?tab=readme-ov-file#similar-projects) like Spacehuhn's deauther.

**Reply by SAQO101** [reply]
OR PLS MAKE A esp32 this project you made "capturing handshakes" 

To not to make a wifi thats a problem 
Can you make it to work like this ?

Connect to specific wifi that i gave to it 
Then i can connect to they esp32 tool 
And control it from broser 
Or cli I REALLY REALLY  need it to work over they cli 

Pls i cant afford or get they wifi adapter any kind that suports a monitor and packet injection modes 


So i use microcontrollers pls help pls

Btw im cyber sec. Student and im suck at this process like well long time 

Pls pls veryy please make they project possible 

That could work over they cli !

That we can control it using codes
Or something pls this project is very cool
